agents/entities/SiteTypes_entries_en.json

```diff
@@ -80,5 +80,29 @@
       "data centre",
       "data center"
     ]
+  },
```

Comment:
Ooookay, I see what you mean when you said pie and sheep. I don't think I ever added anything here, or ever used this type. I just use `@sys.any` to capture arbitrary text (which probably comes with its own problems, but oh well).